recipes/paraview/6.0.1/gh200/post-install

```diff
@@ -0,0 +1,155 @@
+SECTION "configure paraview build..."
+mkdir -p $PARAVIEW_BINARY_DIR && cd $PARAVIEW_BINARY_DIR
+
+cmake -G Ninja \
+    -DCMAKE_INSTALL_PREFIX=$PARAVIEW_INSTALL_DIR            \
+    -DCMAKE_BUILD_TYPE=Release                              \
+    -DPARAVIEW_BUILD_EDITION=CANONICAL                      \
+    -DPARAVIEW_BUILD_TESTING=OFF                            \
+    -DPARAVIEW_BUILD_WITH_EXTERNAL=ON                       \
+    -DPARAVIEW_INSTALL_DEVELOPMENT_FILES=ON                 \
+    -DPARAVIEW_USE_CUDA=ON                                  \
+    -DCMAKE_CUDA_ARCHITECTURES=90a                          \
+    -DPARAVIEW_USE_MPI=ON                                   \
+    -DPARAVIEW_USE_PYTHON=ON                                \
+    -DPARAVIEW_USE_QT=OFF                                   \
+    -DPARAVIEW_USE_VISKORES=ON                              \
+    -DPARAVIEW_ENABLE_RAYTRACING:BOOL=ON                    \
+    -DPARAVIEW_ENABLE_CATALYST=ON                           \
+    -DPARAVIEW_ENABLE_WEB:BOOL=OFF                          \
+    -DPARAVIEW_ENABLE_ADIOS2=ON                             \
+    -DPARAVIEW_ENABLE_FIDES:BOOL=ON                         \
+    -DPARAVIEW_ENABLE_VISITBRIDGE:BOOL=ON                   \
+    -DPARAVIEW_PLUGIN_ENABLE_CDIReader=ON                   \
+    -DPARAVIEW_PLUGIN_ENABLE_NetCDFTimeAnnotationPlugin=ON  \
+    -DPARAVIEW_PLUGIN_ENABLE_pvNVIDIAIndeX=ON               \
+    -DPARAVIEW_PLUGIN_AUTOLOAD_pvNVIDIAIndeX=OFF            \
+    `# VTK options`                                         \
+    -DVTK_USE_X=OFF                                         \
+    -DVTK_OPENGL_HAS_EGL=ON                                 \
+    -DVTK_SMP_IMPLEMENTATION_TYPE=TBB                       \
+    -DVTK_MODULE_USE_EXTERNAL_VTK_token=OFF                 \
+    -DVTK_MODULE_USE_EXTERNAL_VTK_exprtk=OFF                \
+    -DVTK_MODULE_USE_EXTERNAL_VTK_vtkviskores=OFF           \
+    -DVTKOSPRAY_ENABLE_DENOISER:BOOL=ON                     \
+    $PARAVIEW_SRC_DIR/
+
+# =====================================
+SECTION "build and install paraview"
+ninja install
+
+# =====================================
+SECTION "build gadget plugin"
+git clone https://github.com/jfavre/ParaViewGadgetPlugin $SRC_ROOT/ParaViewGadgetPlugin
+mkdir $BUILD_ROOT/ParaViewGadgetPlugin
+cd    $BUILD_ROOT/ParaViewGadgetPlugin
+
+cmake -G Ninja \
+    -DCMAKE_PREFIX_PATH=$PARAVIEW_INSTALL_DIR \
+    -DCMAKE_INSTALL_PREFIX=$PARAVIEW_PLUGINS_DIR \
+    $SRC_ROOT/ParaViewGadgetPlugin
+ninja install
+
+# =====================================
+SECTION "set permissions"
+chmod -R a+r $PARAVIEW_INSTALL_DIR
+chmod -R a+r $PARAVIEW_PLUGINS_DIR
+
+# =====================================
+# TODO check if we can avoid this and every user have to setup their own license
+SECTION "download nvindex license file"
+curl -L "https://jfrog.svc.cscs.ch/artifactory/paraview/nvindex_config.xml" -o $PARAVIEW_PLUGINS_DIR/nvindex_config.xml
+
```

Comment:
```suggestion
```

I believe this license is personal and we should suggest every user to download their own.

They can either put it in the default location or just point to it with an environment variable.

> nvindex_config.xml: NVIDIA IndeX for ParaView Plugin configuration file. The plugin will look for this configuration file in the default ParaView configuration directory (typically $HOME/.config/ParaView/) or, if it is defined, in the directory specified by the environment variable NVINDEX_PVPLUGIN_HOME.